code-challenges.js

```diff
@@ -9,40 +9,149 @@
 // --------------------1) Create a function that determines if a temperature is below boiling point, at boiling point or above boiling point. Boiling point is 212 degrees Fahrenheit.
 // Use the test variables provided below. Expected outcome: "85 is below boiling point" "350 is above boiling point" "212 is at boiling point"
 
+
+// I will need to create conditions in the function that will give me three different outcomes
+
 var temp1 = 85
 var temp2 = 350
 var temp3 = 212
+var temp4 = "pineapple"
+
+//Create a function called isBoiling
+//takes in one argument (temp)
+const isBoiling = (temp) =>{
```

Comment:
great naming! functions that begin with `is` generally denote that it will return a boolean but this is still good practice!